### ISSUE TITLE
Updating to new version requires reboot

### ISSUE THREAD

**Issue by MusiqueGraeme** [bleachbit/bleachbit#1464]
When I just updated from 4..4.0 to 4.4.2 I had to reboot my computer after UNinstalling 4.4.0. **before** installing 4.4.2 OK this may not look like a "bug" but I'm trying to automate the deployment of BleachBit and it makes it a messy process. 

**Expected behavior**
Reboot - if necessary - should happen after the installation of the new version is complete.

**Desktop (please complete the following information):**
 - Windows 11
 - BleachBit version 4.4.2

**Comment by az0** [comment]
@MusiqueGraeme 

>When I just updated from 4..4.0 to 4.4.2 I had to reboot my computer after UNinstalling 4.4.0. before installing 4.4.2 

So it went like this

1. You had 4.4.0 installed
2. You started the 4.4.2 installer
3. The 4.4.2 installer started the 4.4.0 uninstaller
4. The system rebooted
5. You started the 4.4.2 installer again
6. The installation finished normally

Is that right?

> I'm trying to automate the deployment of BleachBit and it makes it a messy process.

When this happened, were you doing a [silent install](https://docs.bleachbit.org/doc/install-on-windows.html)?

**Comment by MusiqueGraeme** [comment]
Yes, that's what happened.. No, I don't tend to do a silent install... maybe I could but what would happen to the reboot?

Blessings
Graeme

Sent with [Proton Mail](https://proton.me/) secure email.

------- Original Message -------
